uefi/src/main.rs

```diff
@@ -136,7 +136,7 @@ fn main_inner(image: Handle, mut st: SystemTable<Boot>) -> Status {
         }
     );
 
-    let mmap_storage = {
+    let _mmap_storage = {
```

Comment:
This is no longer needed because of https://github.com/rust-osdev/uefi-rs/pull/653. We should remove this mapping, not just silence the dead code warning.